bftengine/src/communication/PlainTcpCommunication.cpp

```diff
@@ -931,11 +926,14 @@ class PlainTCPCommunication::PlainTcpImpl {
 
   virtual ~PlainTcpImpl() {
     LOG_TRACE(_logger, "PlainTCPDtor");
+    _pIoThread = nullptr;
```

Comment:
its done only for being very explicit on the destruction. you are right, it doesn't add anything to the destruction. I will remove it to make it less confusing - but just in case - I always prefer to do things explicitly, even if they are meaningless somehow. It gives better semantics sometimes.